src/fenced/fenced/disks.py

```diff
@@ -67,7 +67,7 @@ def _run_batch(self, method, args=None, disks=None, done_callback=None):
             done_notdone = fut_wait(fs.keys(), timeout=30)
         finally:
             executor.shutdown(wait=False)
-        failed = {fs[i] for i in done_notdone.not_done}
+        failed = [fs[i] for i in done_notdone.not_done]
```

Comment:
Why are we changing from a set to a list?

Reply:
Mistake when pushing my local changes when I was debugging.

Reply:
I fixed it.